src/lib/mina_base/parties.ml

```diff
@@ -908,11 +915,44 @@ let weight (parties : t) : int =
     ]
 
 module Valid = struct
-  module Stable = Stable
+  module S = Stable
+
+  module Verification_key_hash = struct
+    [%%versioned
+    module Stable = struct
+      module V1 = struct
+        type t = Zkapp_basic.F.Stable.V1.t
+        [@@deriving sexp, compare, equal, hash, yojson]
+
+        let to_latest = Fn.id
+      end
+    end]
+
+    [%%define_locally Stable.Latest.(equal)]
```

Comment:
I think this is not needed, the type `t` will automatically have `deriving equal` generated.